section/doc/tactic-setup/setup/advanced-trigger-setup/index.html

```diff
@@ -1045,47 +1087,92 @@ <h2 id="_advanced_trigger_setup">Advanced Trigger Setup</h2>
+    var name = input.display;
+    var name_id =input.value;
+    if (!name) input.value = null;
+    var form_table = input.firing_element.getParent(".spt_form_table");
+    var last_input_name = form_table.getElement('.spt_last_input_name');
+    var last_names = last_input_name.value.split('||');
+    // Check if the current name input is what was typed in last
+    var reused_name = false;
+    // Check if the current name is unknown in my_sType4 table
+    var unknown_name = false;
+    if (last_names.length == 2)
+    {
+        reused_name = last_names[0] == name;
+        unknown_name = (last_names[1] == name_id ) &amp;&amp; (last_names[0] != name);
+    }
+    if (!name || name_id == '' || name_id === null || reused_name || unknown_name)
+    { }
+    else {
+        var kwargs = { 'view': 'my_project.name_detail.popup',
+        'name_id': name_id };
+        // Load the popup
+        var pop = spt.panel.load_popup("Name Details", class_name, kwargs);
+        last_input_name.value = name + '||' + name_id;
+        pop.activator = form_table;
+    }
+}
+catch( e )  {
+    spt.alert(spt.exception.handler( e ));
+}</code></pre>
+</div></div>
+</li>
+<li>
+<p>
+In order to have a lookahead that will look at the my_sType4 table,
 a lookahead must be built within a custom view i.e.
 my_views/lookahead_form in the Custom Layout Editor (this will be a
 different view from the aforementioned name details form). An example of
-the code required to create this view is shown below.</p></div>
-<div class="paragraph"><p><em>HTML Tab in Custom Layout Editor for New View</em>&lt;div&gt; &lt;div&gt; &lt;% name_key =
-kwargs.get("search_key") or "" %&gt; &lt;/div&gt; &lt;table&gt; &lt;tr&gt; &lt;td&gt;Name&lt;/td&gt; &lt;td&gt;
-&lt;element name="name"&gt; &lt;display
-class="tactic.ui.input.LookAheadTextInputWdg"&gt;
-&lt;search_type&gt;my_project/my_sType4&lt;/search_type&gt; &lt;column&gt;name&lt;/column&gt;
-&lt;search_key&gt;${name_key}&lt;/search_key&gt;
-&lt;filter_search_type&gt;metadata/Title&lt;/filter_search_type&gt;
-&lt;value_column&gt;id&lt;/value_column&gt;
-&lt;current_value_column&gt;id&lt;/current_value_column&gt; &lt;/display&gt; &lt;/element&gt;
-&lt;/td&gt; &lt;/tr&gt; &lt;/table&gt; &lt;/div&gt;
-5.  Go to the my_views/lookahead_form in Custom Layout Editor and click
+the code required to create this view is shown below.
+</p>
+<div class="paragraph"><p><em>HTML Tab in Custom Layout Editor for New View</em></p></div>
+<div class="listingblock">
+<div class="content">
+<pre><code>&lt;div&gt;
+&lt;div&gt;
+&lt;% name_key = kwargs.get("search_key") or "" %&gt;
+&lt;/div&gt; &lt;table&gt; &lt;tr&gt; &lt;td&gt;Name&lt;/td&gt; &lt;td&gt;
+&lt;element name="name"&gt;
+    &lt;display class="tactic.ui.input.LookAheadTextInputWdg"&gt;
+        &lt;search_type&gt;my_project/my_sType4&lt;/search_type&gt; &lt;column&gt;name&lt;/column&gt;
+        &lt;search_key&gt;$\{name_key}&lt;/search_key&gt;

```

Comment:
I removed it.. it was put in unnecessarily by the auto-conversion from before.. thanks